fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/loan/LoanImportHandler.java

```diff
@@ -431,7 +431,7 @@ private Integer importDisbursalData(CommandProcessingResult result, int rowIndex
             String linkAccountId = disbusalData.getLinkAccountId();
             GsonBuilder gsonBuilder = new GsonBuilder();
             gsonBuilder.registerTypeAdapter(LocalDate.class, new DateSerializer(dateFormat));
-            if (linkAccountId != null && linkAccountId != "") {
+            if (linkAccountId != null && !"".equals(linkAccountId)) {
```

Comment:
I am late again :( 
@percyashu  @vorburger I haven't seen "".equals being used before, should we not use linkAccountId.isEmpty() instead? Although it seems they mean the same thing.

Reply:
that would indeed be a little nicer, yes (I merged it anyway just to move forward, and because it was a "correct" fix in the "mechanical" sense)